test/data/osbuild-composer.toml

```diff
@@ -14,3 +15,4 @@ keytab = "/etc/osbuild-composer/client.keytab"
 [worker]
 allowed_domains = ["localhost", "client.osbuild.local"]
 ca = "/etc/osbuild-composer/ca-crt.pem"
+jwt_tenant_provider_fields = ["rh-org-id"]
```

Comment:
I don't think local worker + jwt works. You need a remote rework + all the jwt related config similar to the koji config section.